apps/frontend/src/app/2_molecules/DatePicker/DatePicker.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FC, useCallback, useState } from 'react';
+
+import classNames from 'classnames';
+import dayjs from 'dayjs';
+import { DayPicker } from 'react-day-picker';
+
+import { Icon, IconNames, Paragraph } from '@sovryn/ui';
+
+import styles from './DatePicker.module.css';
+
+type DatePickerProps = {
+  date: number;
+  onChange: (value: number) => void;
+  className?: string;
+  minDate?: number;
+  maxDate?: number;
+  label: string;
+  isCalendarVisible?: boolean;
+  setIsCalendarVisible: (value: React.SetStateAction<boolean>) => void;
+};
+
+export const DatePicker: FC<DatePickerProps> = ({
+  date,
+  onChange,
+  className,
+  label,
+  minDate,
+  maxDate,
+  isCalendarVisible,
+  setIsCalendarVisible,
+}) => {
+  const [selected, setSelected] = useState<Date | undefined>(
+    dayjs.unix(date).toDate(),
+  );
+
+  const handleDaySelect = (date: Date | undefined) => {
+    if (!date) {
+      return;
+    }
+    setSelected(date);
```

Comment:
There is a problem with having `selected` controlled by the component itself - go to `Open new loan` dialog, select other than the default date in the calendar and click `max duration`. You'll see that the input is updated but if you open the calendar, you'll see the old value selected.
<img width="379" alt="Pasted Graphic" src="https://github.com/DistributedCollective/sovryn-dapp/assets/14878713/7c63c8d1-6aba-4591-8fae-772e7e57de6a">